assets/sass/v2/_identify.scss

```diff
@@ -78,3 +78,20 @@
     }
   }
 }
+
+.o-form-button-bar {
+  .links-container {
```

Comment:
instead of saying container.. change the name to `links-primary` ? It can identify that these links are special then footer links..

Reply:
I just updated this.